extensions/cornerstone-3d/src/commandsModule.js

```diff
@@ -304,11 +322,146 @@ const commandsModule = ({ servicesManager }) => {
       const { viewport } = enabledElement;
       const options = { delta: direction };
 
-      cornerstone3DTools.utilities.stackScrollTool.scrollThroughStack(
+      cornerstoneTools.utilities.stackScrollTool.scrollThroughStack(
         viewport,
         options
       );
     },
+    async createSegmentationForDisplaySet({ displaySetInstanceUID }) {
+      const volumeId = displaySetInstanceUID;
+
+      const segmentationUID = cornerstone.utilities.uuidv4();
+      const segmentationId = `${volumeId}::${segmentationUID}`;
+
+      await cornerstone.volumeLoader.createAndCacheDerivedVolume(volumeId, {
+        volumeId: segmentationId,
+      });
+
+      // Add the segmentations to state
+      cornerstoneTools.segmentation.addSegmentations([
+        {
+          segmentationId,
+          representation: {
+            // The type of segmentation
+            type: cornerstoneTools.Enums.SegmentationRepresentations.Labelmap,
+            // The actual segmentation data, in the case of labelmap this is a
+            // reference to the source volume of the segmentation.
+            data: {
+              volumeId: segmentationId,
+            },
+          },
+        },
+      ]);
+
+      return segmentationId;
+    },
+    async addSegmentationRepresentationToToolGroup({
+      segmentationId,
+      toolGroupId,
+      options,
+    }) {
+      const { representationType } = options;
```

Comment:
If representationType is mandatory, should it really be in `options`? Shouldn't it be a top level argument after `segmentationId` and `toolGroupId`?